app/assets/config/manifest.js

```diff
@@ -0,0 +1,21 @@
+//= link_tree ../images
```

Comment:
Yes, this is an API in sprockets. I don't think multiple manifest files are a thing